src/library_sockfs.js

```diff
@@ -280,6 +280,13 @@ mergeInto(LibraryManager.library, {
 
         function handleMessage(data) {
           assert(typeof data !== 'string' && data.byteLength !== undefined);  // must receive an ArrayBuffer
+
+          // An empty ArrayBuffer will emit a pseudo disconnect event
```

Comment:
I am not sure if a test is really needed as from my point of view this is a very trivial solution and my analysis of module there shouldn't be any side effects.

So think it's okay to merge without a proper test :).